tests/integ/test_horovod_mx.py

```diff
@@ -0,0 +1,114 @@
+# Copyright 2017-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import json
+import os
+import tarfile
+from six.moves.urllib.parse import urlparse
+
+import boto3
+import pytest
```

Comment:
(six is third-party)

```suggestion

import boto3
import pytest
from six.moves.urllib.parse import urlparse
```